src/transactions/TransactionFrame.cpp

```diff
@@ -531,15 +525,7 @@ TransactionFrame::commonValidPreSourceAccountLoad(
         getResult().result.code(txINSUFFICIENT_FEE);
         return false;
     }
-    auto extraSigners = getExtraSigners();
-    for (size_t i = 0; i < extraSigners.size(); i++)
-    {
-        if (!checkSignatureExtraSigner(signatureChecker, extraSigners[i]))
-        {
-            getResult().result.code(txBAD_AUTH);
-            return false;
-        }
-    }
```

Comment:
It looks like this logic for checking signatures was pulled out of the common validity checks? The common validity check function was just renamed. This was actually its old function signature, then it changed just before I forked, and now it seems the core team changed it back. Either way, we just need to pass in the signature checker as an extra parameter and these few lines of code.

Reply:
yes I moved it to `commonValid` because `commonValidPreSeqNum` isn't being given `signatureChecker` anymore. You can see the commit I moved it here: https://github.com/stellar/stellar-core-experimental-cap21and40/pull/2/commits/b2c4515890d024c539e3d1bc4b513ad6e7087a69 (had a TODO reminder there to check with that I later removed bc causing errors)